rocAL/rocAL/include/meta_data/meta_node_flip.h

```diff
@@ -34,7 +34,7 @@ class FlipMetaNode:public MetaNode
 {
     public:
         FlipMetaNode() {};
-        void update_parameters(MetaDataBatch* input_meta_data)override;
+        void update_parameters(MetaDataBatch* input_meta_data, bool segmentation)override;
```

Comment:
Same as previous comments. Make changes wherever applicable

Reply:
Resolved in the latest commit